docusaurus/docs/React/release-guides/new-reactions.mdx

```diff
@@ -0,0 +1,176 @@
+# Introducing new reactions (with breaking changes)
+
+As a "move-away-from-EmojiMart" initiative we decided to start with reaction components (selector and list) and revamp the DX to make customization of these components much easier and more intuitive without having to rebuild default selector and list components from scratch.
+
+### Main reasons for a revamp
+
+- inability to reuse default (Stream supplied reactions) with your custom ones
+- strong reliance on EmojiMart and inability to use completely custom reactions out of the box
+- certain `additionalEmojiProps` did not work with Stream-supplied reactions
```

Comment:
I'd mention smaller bundle size, that's always a nice benefit, and emoji-mart is quite big

Reply:
Unfortunately, we're not getting rid of it completely - yet, there's still parts of the SDK that use it - but that's the idea.